styles/styles.css

```diff
@@ -15,35 +15,653 @@ li {
+  position: relative;
+}
+.chevron-right::after {
+  content: "";
+  display: block;
+  box-sizing: border-box;
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  border-bottom: 2px solid;
+  border-right: 2px solid;
+  transform: rotate(-45deg);
+  left: 4px;
+  top: 2px;
+}
+/* ========================== End Main styles ========================== */
+
+/* ========================== Navigation ========================== */
+.site-nav {
+  overflow: hidden;
+  background-color: var(--primary);
+}
+
+.toggle,
+[id^="drop"] {
+  display: none;
+}
+
+nav {
+  margin: 0;
+  padding: 0;
+  background-color: var(--primary);
+  padding: 0 15%;
+}
+
+#logo {
+  display: flex;
+  align-items: center;
+  margin-left: 0;
+  padding-left: 0;
+  float: left;
+  margin: 30px 0;
+}
+
+#logo span {
+  font-size: 0.8em;
+  font-weight: 700;
+  line-height: 1.2em;
+  margin-left: 10px;
+  text-transform: uppercase;
+  color: var(--primary-font-color);
+}
+
+nav:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+nav ul {
+  float: right;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  position: relative;
+}
+
+nav ul li {
+  margin: 0px;
+  display: inline-block;
+  float: left;
+  background-color: var(--primary);
+}
+
+nav ul li i {
+  margin-left: 5px;
+}
+
+nav a {
+  display: block;
+  padding: 14px 20px;
+  text-decoration: none;
+  color: var(--primary-font-color);
+  font-size: 0.9em;
+  text-transform: uppercase;
+  text-align: left;
+  font-weight: 600;
+}
+
+nav a:hover {
+  background-color: var(--gray-color);
+}
+
+nav ul ul {
+  display: none;
+  position: absolute;
+}
+
+nav ul li:hover > ul {
+  display: inherit;
+  z-index: 1;
+}
+
+nav ul ul li {
+  width: 170px;
+  float: none;
+  display: list-item;
+  position: relative;
+  z-index: 1;
+}
+
+.menu {
+  margin-top: 20px;
+}
+
+.menu-icon > .line {
+  background-color: var(--primary-font-color);
+  height: 2px;
+  display: block;
+}
+.menu-icon > .line + .line {
+  margin-top: 8px;
+}
+
+.main-toggle {
+  width: 40px;
+}
+
+.main-toggle:hover {
+  background: var(--primary);
+}
+
+#burger {
+  width: 40px;
+  height: 20px;
+  position: relative;
+  margin: 10px auto;
+  -webkit-transform: rotate(0deg);
+  -moz-transform: rotate(0deg);
+  -o-transform: rotate(0deg);
+  transform: rotate(0deg);
+  -webkit-transition: 0.5s ease-in-out;
+  -moz-transition: 0.5s ease-in-out;
+  -o-transition: 0.5s ease-in-out;
+  transition: 0.5s ease-in-out;
+  cursor: pointer;
+}
+
+#burger span {
+  display: block;
+  position: absolute;
+  height: 5px;
+  width: 100%;
+  background: var(--primary-font-color);
+  border-radius: 9px;
+  opacity: 1;
+  left: 0;
+  -webkit-transform: rotate(0deg);
+  -moz-transform: rotate(0deg);
+  -o-transform: rotate(0deg);
+  transform: rotate(0deg);
+  -webkit-transition: 0.25s ease-in-out;
+  -moz-transition: 0.25s ease-in-out;
+  -o-transition: 0.25s ease-in-out;
+  transition: 0.25s ease-in-out;
+}
+
+#burger span:nth-child(1) {
```

Comment:
Recommended using relative positioning (top: 50%/100%) instead of absolute (0px, 15px). In this case no need to adjust everything if a control size was changed.